pkg/controller/config/common/extensions.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package common
+
+import (
+	mf "github.com/jcrossley3/manifestival"
+	// servingv1alpha1 "github.com/knative/serving-operator/pkg/apis/serving/v1alpha1"
+	tektonv1alpha1 "github.com/openshift/tektoncd-pipeline-operator/pkg/apis/operator/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Activities []func(client.Client, *runtime.Scheme, *tektonv1alpha1.Config) (*Extension, error)
+type Extender func(*tektonv1alpha1.Config) error
+type Extensions []Extension
+type Extension struct {
+	Transformers []mf.Transformer
+	PreInstalls  []Extender
+	PostInstalls []Extender
+}
+
+func (activities Activities) Extend(c client.Client, scheme *runtime.Scheme, install *tektonv1alpha1.Config) (result Extensions, err error) {
```

Comment:
`install` change this name as our CRD is not `install.tekton.dev` anymore

Reply:
ok, change to `config`?